src/engine/shared/entities/player.lua

```diff
@@ -117,6 +128,10 @@ function player:getName()
 	return self:getNetworkVar( "name" )
 end
 
+function player:getPosition()
+	return self:getNetworkVar( "position" )
+end

```

Comment:
On hold, I've recently deprecated networkvar helpers like this. I don't know how I feel about adding special networkvar helpers back